lib/Traits/TraitsClasses/Cesium3dTilesTraits.ts

```diff
@@ -173,4 +173,12 @@ export default class Cesium3DTilesTraits extends mixTraits(
       "One or many properties of a feature that together identify it uniquely. This is useful for setting properties for individual features. eg: ['lat', 'lon'], ['buildingId'] etc."
   })
   featureIdProperties?: string[];
+
+  @primitiveArrayTrait({
+    name: "lightColor",
+    type: "number",
+    description:
+      "The light color when shading models. When undefined the scene's light color is used instead. eg: [255, 255, 255]."
+  })
+  lightColor?: number[];
```

Comment:
Suggestion: Most other color traits in Terria are typed as a string and then converted to Cesium.Color using [Cesium.Color.fromCSSString()](https://cesium.com/learn/cesiumjs/ref-doc/Color.html?classFilter=color#.fromCssColorString) ([example](https://github.com/TerriaJS/terriajs/blob/ab9e2cf1280428402ece6e5658edfddf2f217944/lib/Traits/TraitsClasses/Table/ColorStyleTraits.ts#L22)). I think we should follow the same pattern here. What do you think?

Reply:
@ykiu - Thanks! That makes sense to me now.

Could you please git pull and run `yarn run prettier` - some lines in CHANGES.md is failing prettier.